.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,69 @@
+name: Run Integration Tests
+
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
+  push:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ["3.8.x"]
+        # TODO: change to 1.0.1 when available
+        dbt: ["1.0.1rc1"]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+
+      - name: Fix jinja
+        run: |
+          pip install markupsafe==2.0.1
+
+      - name: Install fal
+        run: |
+          pip install fal
```

Comment:
I think we don't use this, right?

Reply:
We're using fal to validate model contents by comparing dataframes.